lib/entity_cache/storage/persistent.rb

```diff
@@ -2,10 +2,11 @@ class EntityCache
   module Storage
     module Persistent
       extend Configure::Macro
-      configure :persistent_store
 
       def self.included(cls)
         cls.class_exec do
+          configure :persistent_store
```

Comment:
I believe this is my first glimpse of this in the wild. Pretty cool.

What if it was like:

```ruby
configure :persistent_store do |some_build_arg1, some_build_arg_2|
  # this block is meta-programmed into the class as the build method
  # the above block args are the build method's args
  thing = something(some_build_arg1, some_build_arg2)
  instance = new(thing)
  Logger.configure instance
  # etc
  instance
end
```

Is that crazy?